### ISSUE TITLE
RecursionError in auditwheel show

### ISSUE THREAD

**Issue by anntzer** [pypa/auditwheel#77]
Running auditwheel show on a wheel of https://github.com/anntzer/mpl_cairo compiled with gcc7.1 (which is definitely not manylinux1-compatible, I was just doing some tests) gives the following traceback:
```
Traceback (most recent call last):
  File "/home/antony/.local/bin/auditwheel", line 11, in <module>
    sys.exit(main())
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/main.py", line 49, in main
    rval = args.func(args, p)
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/main_show.py", line 28, in execute
    winfo = analyze_wheel_abi(args.WHEEL_FILE)
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/wheel_abi.py", line 73, in analyze_wheel_abi
    get_wheel_elfdata(wheel_fn)
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/wheel_abi.py", line 56, in get_wheel_elfdata
    ctx.path)
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/policy/external_references.py", line 53, in lddtree_external_references
    whitelist)  # type: List[str]
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/policy/external_references.py", line 39, in get_req_external
    whitelist) for lib in libs), libs)
  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/policy/external_references.py", line 39, in <genexpr>
    whitelist) for lib in libs), libs)

<elided>

  File "/home/antony/.local/lib/python3.6/site-packages/auditwheel/policy/external_references.py", line 39, in <genexpr>
    whitelist) for lib in libs), libs)
RecursionError: maximum recursion depth exceeded while calling a Python object
```

(Python 3.6.1 ArchLinux, auditwheel from pypi).

**Comment by anntzer** [comment]
Kindly bumping the issue?

**Comment by anntzer** [comment]
Actually it was not that hard to solve, see #84.

**Comment by anntzer** [comment]
Closed by #84.
